package.json

```diff
@@ -42,4 +43,4 @@
     "Alexandre Perrin <alex@kaworu.ch> (https://github.com/kAworu)",
     "Daniel Beer <dlbeer@gmail.com> (https://github.com/dlbeer)"
   ]
-}
+}
```

Comment:
nitpicking: Please keep the EOL at the end.

Reply:
I blame the automatic formatting built into VS Code 🤣